index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const imagemin = require('imagemin');
+const imageminMozjpeg = require('imagemin-mozjpeg');
+const imageminPngquant = require('imagemin-pngquant');
+const Svgo = require('svgo');
+const fs = require('fs');
+const os = require('os');
+
+module.exports.compress = (file, level, callback) => {
```

Comment:
Let's just export a single method that takes in the file or stream and some options and it figures out how to process automatically.  We could add support other file types too